### PR TITLE
Google cloud integration example. Box Skills + GCP Vision Product Search

### PR DESCRIPTION
**Pull request by ashokhollav**
An example to showcase Google Cloud Vision Product Search usage to enhance Box files using Box Skills kit

### CODE REVIEW

File: sample-google-product-search-integration/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (cbetta) [inline sample-google-product-search-integration/LICENSE.txt]: Can we remove this file? We don't need it as we have one at the root of the repo.

File: sample-google-product-search-integration/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+This guide demonstrates how to build a Box Skill application in Node.js using Serverless, a command line tool for building and deploying applications to serverless architectures, and the Box Skills Kit, a library for interacting with the Box API and integrate it with AI/ML services from Google Cloud !
```
Comment (cbetta) [inline sample-google-product-search-integration/README.md]: @ashokhollav can you please rewrite this following the format set out in this sample? https://github.com/box-community/sample-image-skills/tree/master/acuant-assureid-goverment-id-data-extraction

File: sample-google-product-search-integration/index.js [hunk 3]
```diff
@@ -0,0 +1,172 @@
+// Import FilesReader and SkillsWriter classes from skills-kit-2.0.js library
+const { FilesReader, SkillsWriter, SkillsErrorEnum } = require('./library/skills-kit-2.0');
+var Set = require("collections/set");
+const vision = require('@google-cloud/vision');
+
+function max(price) {
```
Comment (cbetta) [inline sample-google-product-search-integration/index.js]: Seems to no longer be used?

File: sample-google-product-search-integration/LICENSE.txt (hunk 1 above)
Comment (ashokhollav) [reply sample-google-product-search-integration/LICENSE.txt]: done, removed the file and the link

File: sample-google-product-search-integration/README.md (hunk 2 above)
Comment (ashokhollav) [reply sample-google-product-search-integration/README.md]: done, please review and let me know if its OK now.

File: sample-google-product-search-integration/index.js (hunk 3 above)
Comment (ashokhollav) [reply sample-google-product-search-integration/index.js]: @cbetta , removed the function max(price)

**Pull request merged**